app-text/pdf2djvu/pdf2djvu-0.9.17.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
Old copyright header... when committing with `repoman` it would automatically update this for you.